### PR TITLE
Script to Print Versions of Software on Robot

### PR DESCRIPTION
**Pull request by bcoltin**
For Issue #86

### CODE REVIEW

**Comment by bcoltin** [comment]
@kbrowne15 Can you add the HLP stuff where I put a todo? Thanks!

File: astrobee/scripts/print_version.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Print the software versions on the MLP, LLP, and HLP.
+
+if [ `hostname` != mlp ] ; then
+    echo "Must be run on mlp."
+    exit 1
+fi
+
+echo "=================== MLP ==================="
+./cpu_print_version.sh | sed 's/^/  /'
```
Comment (marinagmoreira) [inline astrobee/scripts/print_version.sh:11]: Does this assume that we also need to be on the folder where the script is?

File: astrobee/scripts/cpu_print_version.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+echo "  astrobee:           $(deb_version astrobee)"
+echo "  astrobee-config:    $(deb_version astrobee-config)"
+echo "  astrobee-avionics:  $(deb_version astrobee-avionics)"
+echo "FSW Version:"
+cat /opt/astrobee/version.txt | sed 's/^/  /'
+echo "Modified Files:"
+dpkg -V astrobee | sed 's/^/  /'
+dpkg -V astrobee-config | sed 's/^/  /'
+dpkg -V astrobee-avionics | sed 's/^/  /'
+#echo "  astrobee: $(dpkg -s astrobee | grep -oP 'Version: \K.+')"
```
Comment (marinagmoreira) [inline astrobee/scripts/cpu_print_version.sh:27]: Remove commented repeated code?

File: astrobee/scripts/print_version.sh (hunk 1 above)
Comment (bcoltin) [reply astrobee/scripts/print_version.sh:11]: Good point!

**Review by marinagmoreira — APPROVED** [review]

**Pull request merged**